kolibri/plugins/device/assets/src/views/DeprecationWarningBanner.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+
+  <div
+    v-show="showBanner"
+    class="alert"
+    :style="{ backgroundColor: $themePalette.yellow.v_100 }"
+  >
+    <div style="display:flex">
+      <div>
+        <KIcon
+          icon="warning"
+          class="icon"
+          :color="$themePalette.amber.v_a400"
+        />
+      </div>
+
+      <div class="error-message">
+        <p v-if="py27Deprecated">
+          {{ coreString('pythonSupportWillBeDropped') }}
+        </p>
+        <p v-if="ie11Deprecated">
+          {{ coreString('browserSupportWillBeDroppedIE11') }}
+        </p>
+      </div>
+    </div>
+  </div>
+
+</template>
+
+
+<script>
+
+  import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
+  import plugin_data from 'plugin_data';
+
+  export default {
+    name: 'DeprecationWarningBanner',
+    mixins: [commonCoreStrings],
+    computed: {
+      showBanner() {
+        return this.ie11Deprecated || this.py27Deprecated;
+      },
+      ie11Deprecated() {
```

Comment:
It seems this got skipped from the design, but we scoped strings to handle two scenarios:
1. The current browser you are using is IE11
2. Users of this Kolibri are using IE11

At the moment, we are only detecting (2), but the displayed message seems more like (1) is happening to the user.

I have IE11 users on my device, so I see this, in Chrome:

![Screenshot from 2023-06-08 13-55-33](https://github.com/learningequality/kolibri/assets/1680573/d0b2f34a-6fc6-4594-b575-2988480e765b)

We should add an extra paragraph in the IE11 warning, conditionalized on whether there are detected IE11 users on the device, or whether the current browser is IE11. If the former, we should display this above the current message: https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/mixins/commonCoreStrings.js#L1297 if the former, we should display this one: https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/mixins/commonCoreStrings.js#L1292